platform/translation_process/translation_states.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: translation_states
+title: Understanding Translation States
+sidebar_label: Translation States
+description: Tolgee organizes translations into three states, each visually shown by a colored bar near the translated key.
+---
+
+import { ScreenshotWrapper } from '../shared/_ScreenshotWrapper';
+
+Translation management is important when you work in a big team and want to ensure that when you translate text, the quality meets your standards. Changing the status of each translation helps teams work together well. You might want to approve or flag a translation as ready. Tolgee has a feature to simplify the process of approval.
+
+## The Three Translation States
+
+Tolgee organizes translations into three states, each visually shown by a colored bar near the translated key:
+
+- **Gray**: Untranslated
+- **Yellow**: Translated but not reviewed
+- **Green**: Reviewed
+
+<ScreenshotWrapper
+  src="/img/docs/platform/translation-states-example.png"
+  alt="Quality assurance in translation three states"
+/>
+
+The color-coding feature helps managers and translators to check the status of a translation. This system makes it visual and easy to go through, especially if you are working with a large number of keys.
+
+## Working with Translation States
+
+To see the translation keys and the translations, go to the translation tab in Tolgee. You will see translations in your primary language. To show more languages of your project, you can do the following:
+
+1. Click on the language bar at the top right that shows the currently selected languages
+2. Select the project languages that you want to see and check
+3. Now you can view all the translations and work with the states
+
+### Quality Assurance in Translation
+
+The reviewer of the project usually performs the following functions with Tolgee:
+
+- **Approving**: To change a translation from "Translated" to "Reviewed" status, click the "Mark as Reviewed" button at the top right corner of the translation.
+- **Reverting Approvals**: If a previously approved translation needs changes, use the same button to change it back to "Translated" status
+- **Comments**: Use comments to provide feedback or explain why it was not approved
+- **Tracking Changes:** Go to Edit to see the history of changes
```

Comment:
"Go to Edit", is suddenly changing the explanations to some king of "how to" form. 
```suggestion
- **Tracking Changes:** See the string history in bottom part of the side panel when translation is open for editing
```